op-proposer/proposer/driver.go

```diff
@@ -192,7 +198,13 @@ func (l *L2OutputSubmitter) FetchNextOutputInfo(ctx context.Context) (*eth.Outpu
 func (l *L2OutputSubmitter) fetchOutput(ctx context.Context, block *big.Int) (*eth.OutputResponse, bool, error) {
 	ctx, cancel := context.WithTimeout(ctx, l.Cfg.NetworkTimeout)
 	defer cancel()
-	output, err := l.RollupClient.OutputAtBlock(ctx, block.Uint64())
+
+	rollupClient, err := l.RollupProvider.RollupClient(ctx)
+	if err != nil {
+		l.Log.Error("proposer unable to get rollup client", "err", err)
```

Comment:
Seeing this, I realize that the other `l.Log.Error` calls in this function (and possibly elsewhere) don't use the right format, whereas you use it correctly here. We should fix them to use the pattern `l.Log.Error("msg", "key1", value1, "key2", value2)` in a follow-up PR.

Reply:
Will do after #8585

Reply:
Actually, this was really quick to do, opening shortly